src/Candidate.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+const url =
+  'http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote?';
+
+function CandidateVotes({ candidate, flipVoteFlag, rank }) {
+  function handleVoteButtonClick() {
+    axios
+      .get(url, {
+        params: {
+          id: candidate.id,
+        },
+      })
+      .then(function (response) {
+        alert(response.data);
+        flipVoteFlag();
```

Comment:
```suggestion
      .then((response) => {
        alert(response.data);
        flipVoteFlag();
```

이왕이면 깔끔한 화살표 함수를 써봅시다 ㅎㅎ